### DISCUSSION TITLE
凱比戰隊

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#626] [General]
AI機器人以深入教育內部，希望可以提升教育品質!
https://news.ltn.com.tw/news/life/breakingnews/4172473

**Comment by awehao** [comment]
同學看到好多你的文章呢，一開始我看到這個凱比戰隊的標題，還以為又是什麼遊戲之類的戰隊，結果是很多凱比組成的戰隊阿，雖然機器人這種新奇有趣的東西確實能夠吸引他們，但還是希望它們可以好好得到教師們的關愛
